contents/founders/write-great-job-ads.md

```diff
@@ -0,0 +1,144 @@
+
+- Competitive salary - ooh let's pay money in exchange for work, how brave
+
+- The legal minimum number of days holiday or pension contributions
+
+- A fast-paced/dynamic/exciting environment with potential for growth/learning/flexibility
+
+- An opportunity to work with great people (please find me the company offering an opportunity to work with extremely average people)
+
+- Describing any kind of policy as 'generous' without specifying how
+
+Instead, be specific for the standard stuff, or be unusual. It is better to have a list of 5 cool things than 10 generic ones. 
+
+## Non-obvious ways to write a great job ad
+
+**Get the hiring manager to write it, and then a good writer to edit it**
+
+Classic process is hiring manager comes up with a list of bullets -> passes to recruiter to write the ad -> publish. This is a great way to ensure that you have a weird Frankenstein's monster of an ad written by two people, neither of whom are professional writers. 
+
+I'm a big believer that a well-written job ad is just as important to your company as a great blog post or homepage copy. Especially if you advertise on LinkedIn, your ad will be seen by _thousands_ of people - potentially more than visit your website every week. 
+
+Instead, do this:
+
+- Hiring manager write the _entire_ job spec. They can copy previous ads. It is ok if they are not a great writer - they just need to write down as much as possible.
+- Get a good writer to edit the ad and make it a _good piece of written content._ If your hiring manager is already a good writer, you can skip this step.
+- Recruiter does a hygiene check for dumb stuff or anything obviously missing.
+
+**Be genuine, or even weird**
+
+Job applicants are sometimes reading dozens of ads in the course of their search, so cutting through the noise is really important - I'm assuming you don't already have some incredibly well-known brand like Google etc. 
+
+Ideally, write in the tone of voice of your company. Tell a story. Your ads don't have to follow the exact same structure every time - engineers care about different things to designers. Sometimes you will want to get super specific about exactly what you want the person to do. Other times you may have a vague sense, but you need help figuring out the role - both of these are ok, but the ads should be written differently. 
+
+I think it's actually ok to write quite a lot of content here. 'Best practice' is to make job specs as short as possible because the internet has wrecked our attention spans, but I think that just optimizes for total application volume over quality. The right candidate will spend lots of time reading and re-reading your spec because it's full of all the things they care about, and in turn you'll probably get fewer, but higher quality, applications. 
```

Comment:
```suggestion
I think it's actually ok to write quite a lot here. 'Best practice' is to make job specs as short as possible because the internet has wrecked our attention spans, but I think that just optimizes for total application volume over quality. The right candidate will spend lots of time reading and re-reading your spec because it's full of all the things they care about, and in turn you'll probably get fewer, but higher quality, applications. 
```